### DISCUSSION TITLE
Any tips on fixing alignment problems

### DISCUSSION THREAD

**Discussion by utunga** [coqui-ai/TTS#654] [General Q&A] [answered]
Hello ! Training a model using tacotron. Pretty much just first crack at this (with coqui) but wondering if there are tips on avoiding alignment problems. 

Voice sounds vaguely human after a few hundred epochs, but alignment is looking like this and I think this is reflected in the sound...

<img width="757" alt="image" src="https://user-images.githubusercontent.com/166867/125428181-2993dd4e-a09f-443b-b9f6-72b6807d22bf.png">

Just wondering if anyone made a checklist or something of common things to check with regards to bad alignment? 

--

My audio data has been manually broken up (by humans) into utterances. (I have about 750 or so utterances total at the moment, but expecting more.)

One thing that i could do - though it would take a bit of work - is to run the whole data set through another model i already have that would allow me to clip the utterances more precisely around the spoken words. Would that be likely to help? (It's a non trivial extra step). 

Alternatively I am wondering if I should maybe turn *off*  the 'do_trim_silence' - these recordings are pretty good quality and made in a studio so perhaps 'trim silence' is not necessary? Current settings as follows 

<img width="524" alt="image" src="https://user-images.githubusercontent.com/166867/125429201-dfa316cc-6eb7-4053-96b3-1e0298993711.png">

**Comment by utunga** [comment]
Any help much appreciated in advance.

**Comment by erogol** [comment, marked as answer]
https://tts.readthedocs.io/en/latest/faq.html

**Reply by utunga** [reply]
fwiw, the problem was that when I cloned the repo to add our dataset I accidentally /stupidly forked from the 0.0.15 tag (or something) instead of 0.1.x .. (bigger number bias) .. once I had the latest version of everything it all made a lot more sense

**Reply by kin0303** [reply]
Hi, have you got the answer?
